src/app/frontend/replicationcontrollerlist/replicationcontrollercard.html

```diff
@@ -111,7 +111,7 @@
           <span flex="initial" class="kd-replicationcontroller-card-section-title">External endpoint</span>
           <div flex>
             <div ng-repeat="endpoint in ::ctrl.replicationController.externalEndpoints track by $index">
-              <kd-service-endpoint endpoint="::endpoint"></kd-service-endpoint>
+              <kd-service-endpoint endpoint="::endpoint" url="true"></kd-service-endpoint>
```

Comment:
Add an implementation note here that says why we always show external endpoints as links. Basically the same as you did in the PR comment. Use `<!-- .... --!>`